x/auth/module.go

```diff
@@ -49,8 +81,17 @@ func (a AppModule) NewQuerierHandler() sdk.Querier {
 }
 
 // module init-genesis
-func (a AppModule) InitGenesis(_ sdk.Context, _ json.RawMessage) ([]abci.ValidatorUpdate, error) {
-	return []abci.ValidatorUpdate{}, nil
+func (a AppModule) InitGenesis(ctx sdk.Context, data json.RawMessage) []abci.ValidatorUpdate {
```

Comment:
can we rename these variables to make them easier to read ?
```suggestion
func (appModule AppModule) InitGenesis(ctx sdk.Context, data json.RawMessage) []abci.ValidatorUpdate {
```

Reply:
hmm - I think that this is pretty serpurflous all these functions are quite small, so the variable `a` should be sufficient. Like for example - with all the keepers (which are much larger functions) we simply use `k` - now that being said I would be cool with making `a` -> `am`

Reply:
Yeah receivers should be concise: `am`.

Reply:
These are added in the culminating branch